demonstrations/tutorial_qft_arithmetics.py

```diff
@@ -66,9 +70,9 @@
 
 The first thing we need to know is PennyLane's
 standard for encoding numbers in binary. A binary number can be
-represented as a string of :math:`1`s and :math:`0`s, which we can represent as the multi-qubit state
+represented as a string of :math:`1` s and :math:`0` s, which we can represent as the multi-qubit state

```

Comment:
```suggestion
represented as a string of :math:`1`s and :math:`0`s, which we can represent as the multi-qubit state
```
Should be without the spaces.